docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,258 @@
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Enable compression of log data?
+
+``http-client.log.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``false``
+
+Enable the logging protocol?
+
+``http-client.log.flush-interval``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``10s``
+
+Set frequency of log flushing to disk?
+
+``http-client.log.max-history``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``15``
+
+Set the maximum file size of a history log?
+
+``http-client.log.max-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-data-size`
+* **Default value:** ``1GB``
+
+Set the maximum size of log file?
+
+``http-client.log.path``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** ``var/log/``
+
+Sets the name of the log file. The log file will have the prefix of the HTTP
+client (<client_name>http-client.log)
+
+``http-client.log.queue-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1``
+* **Default value:** ``10_000``
+
+Set file size for logging queued connections?
+
+``http-client.max-connections``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``200``
+
+Set the maximum connections allowed to the service.
+
+``http-client.max-connections-per-server``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1``
+* **Default value:** ``20``
+
+Set the maximum connections allowed per server.
+
+``http-client.max-content-length``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-data-size`
+* **Default value:** ``16MB``
+
+Set the maximum allowed size of the HTTP request/response.
+
+``http-client.max-requests-queued-per-destination``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Minimum value:** ``1``
+* **Default value:** ``1024``
+
+Set the maximum number of queued connections allowed before failure.
+
+``http-client.record-request-complete``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Record request traffic?
+
+``http-client.request-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Minimum value:** ``0ms``
+* **Default value:** ``5m``
+
+Set the timeout value for the overall request.
+
+``http-client.secure-random-algorithm``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+
+Specifies which secure random algorithm is used to secure the connection. The
+default varies by operating system. Algorithms are specified according to
+standard algorithm name documentation.
+
+?Possible types include ``NativePRNG``, ``NativePRNGBlocking``,
```

Comment:
I'm not sure here but IIRC it's platform and configuration dependent. @electrum wdyt?